src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/caseupdate/CaseUpdateClient.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.HttpStatusCodeException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.CaseClientServiceException;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.ServiceResponseParser;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.model.request.ExceptionRecord;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate.model.request.CaseUpdate;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate.model.request.ExistingCaseDetails;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Wrong lexicographical order for 'uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate.model.request.ExistingCaseDetails' import. Should be before 'uk.gov.hmcts.reform.bulkscan.orchestrator.client.model.request.ExceptionRecord'.](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4535703)